src/Microsoft.OData.Core/UriParser/Aggregation/AggregateTransformationToken.cs

```diff
@@ -0,0 +1,63 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="AggregateTransformationToken.cs" company="Microsoft">

```

Comment:
>AggregateTransformationToken [](start = 20, length = 28)

The renamed file names/classes are very confusing. Should this code be in "AggregateToken.cs", and the code in AggregateToken.cs be in "AggregateExpressionToken.cs"?